### ISSUE TITLE
Option "comp-lzo" is deprecated in OpenVPN 2.4

### ISSUE THREAD

**Issue by winfried-van-loon** [Nyr/openvpn-install#430]
According to Tunnelblick on MacOS, [comp-lzo](https://community.openvpn.net/openvpn/wiki/DeprecatedOptions#a--comp-lzo) is deprecated in OpenVPN 2.4 and may be removed in OpenVPN 2.5.

<img width="617" alt="screen shot 2018-03-07 at 09 46 34" src="https://user-images.githubusercontent.com/756956/37082861-f7c8c81e-21ed-11e8-9f82-d6423b974862.png">

Having this option in the configuration may lead to clients not able to connect to the server. To fix this, it seems we can simply remove the `comp-lzo` option from the configuration.

**Comment by Nyr** [comment]
`comp-lzo` is indeed deprecated and was referenced in #220. I am hoping to switch directly from `comp-lzo` to `compress lz4`.

There is no reason to remove `comp-lzo` prematurely because as far as I know it will not be removed in OpenVPN 2.5, I don't understand why Tunnelblick developers say this but I can't find a source about it. Please let me know if I have missed something.

People should consider that `comp-lzo` is a very widely used option and LZO compression itself isn't being removed or deprecated, only the option has been renamed to a better syntax. That's why I seriously doubt that `comp-lzo` will disappear in 2.5, it would be a very bad idea from my point of view.

**TL;DR:** developers haven't announced the removal of `comp-lzo`. I am well aware that it has been deprecated, but is there for compatibility.

**PS:** you can avoid that warning switching from `comp-lzo` to `compress lzo` in your client.

**Comment by winfried-van-loon** [comment]
Hi! Apologies, while I had opened and read that issue, I had not seen the line about `lzo`. I'm not sure what their source is for showing it to be removed in 2.5. A lot stuff seems to have changed in 2.5, perhaps they've simply made a mistake. 

Would changing `comp-lzo` to `compress lzo` (which helped, thanks!) have any disadvantages if we made this the default?

**Comment by Nyr** [comment]
> Would changing `comp-lzo` to `compress lzo` (which helped, thanks!) have any disadvantages if we made this the default?

Yes, that would break configuration in OpenVPN <2.4. That’s the case for example in the Ubuntu 16.04 official repos.

**Comment by pixelrebel** [comment]
What is the equivalent to `comp-lzo no`?

**Comment by Nyr** [comment]
@pixelrebel read the issue, it is explained in my first response.

**Comment by pixelrebel** [comment]
@Nyr I don't see a mention of how to disable compression with the new syntax.  Reading the specs, it seems there is not a `no` or `none` option.  That would have me believe to disable compression from the client side just `compress` would suffice (which is extremely confusing).

**Comment by Nyr** [comment]
@pixelrebel from the OpenVPN manual:

> If the algorithm parameter is empty, compression will be turned off, but the packet framing for compression will still be enabled, allowing a different setting to be pushed later.

Compression is fully disabled if you don't set `compress` to begin with.

**Comment by pixelrebel** [comment]
Ah okay, I read that as compression is still enabled at some level.  Thanks for the clarification @Nyr !

**Comment by binco** [comment]
i used "compress lzo",but the prompt "compress lzo" is deprecated in OpenVPN 2.5,I consider not compressing
https://superuser.com/questions/1686651/why-is-compression-not-recommended-in-openvpn
